lib/dfe/reference_data/itt/subjects.rb

```diff
@@ -530,6 +555,9 @@ module ITT
                                                       hecos_code: '100069',
                                                       register_category: 'e46e935b-d692-431f-bf0d-b0aad57b6035',
                                                       publish_category: '13',
+                                                      incentive: {
+                                                        '2025-2026' => 'bf92df81-2584-4e94-96a5-5b72fedb3fd6'
+                                                      },
```

Comment:
This code links to languages and not drama

Reply:
This code links to drama 3c038bba-ab48-4b14-8936-f7e2d61019fd - but not sure it's needed as there is no bursary any more

Reply:
Good spot, removed now anyway